ui-src/src/cells/holo-chat/components/channels/newChannel.tsx

```diff
@@ -40,15 +40,16 @@ const styles: StyleRulesCallback = (theme: Theme) => ({
   }
 })
 
-interface Props {
-  classes: any,
+interface OwnProps {
+  classes?: any,
```

Comment:
We should probably apply this change across all of the components. Might make the tests a bit neater